.github/workflows/ci_cd.yml

```diff
@@ -255,7 +255,8 @@ jobs:
   publish-docs:
     name: Publish developer docs
     runs-on: ubuntu-latest
-    if: github.event_name == 'push' && github.repository == 'WordPress/openverse-api'
+    # Check if the event is not triggered by a fork or dependabot
+    if: github.event.pull_request.head.repo.full_name == github.repository && github.actor != "dependabot[bot]"
```

Comment:
Hmm, I'm worried that this won't run if the PR is pushed to (since it's referencing `event.pull_request` vs `event_name == 'push'`. Do you know if there might be a way to test it? We may just want to add `github.actor != "dependabot[bot]"` to the end of the existing conditional.